DuckDuckGoTests/DaxDialogTests.swift

```diff
@@ -18,7 +18,6 @@
 //
 
 import XCTest
```

Comment:
Apologies for polluting the diff with seemingly unrelated changes, but I noticed we had two sets of dax dialog tests left over from the fire button experiment, so I either needed to fix the extra ones to cope with the changes to DetectedTracker, or clean them up, so went for the latter